src/main/java/wellnus/reflection/ExitCommand.java

```diff
@@ -73,8 +76,9 @@ protected String getSupportedCommandArguments() {
     public void execute() {
         try {
             validateCommand(this.argumentPayload);
-        } catch (BadCommandException invalidCommandException) {
-            UI.printErrorFor(invalidCommandException, INVALID_COMMAND_NOTES);
+        } catch (BadCommandException invalidCommand) {
+            LOGGER.log(Level.SEVERE, INVALID_COMMAND_MSG, invalidCommand);
```

Comment:
Hmm the log severity conflicts with what @nichyjt uses for his logging, I believe a lower severity level is more appropriate? Since our app can just ignore an invalid command and still run as normal without taking any measures, making it a trivial issue.

Reply:
Yeah I agree with this line of reasoning. Not important enough to be included together with other breaking exceptions' Java logs

Reply:
> Yes. WARN and above severity should be reserved for _breaking exceptions_ that we choose to catch rather than assert.
> 
> The try catch in this context is to catch for cheeky user input, - it should be noted down for debug purposes as `INFO` for us as a visual confirmation that the code works as intended.

I'm a bit confused here for 'breaking exeptions we choose to catch', cause this is an exception we try to catch?